src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write your code here
+const allPop = document.querySelectorAll('.population');
+const arrPop = [...allPop].map(number => +number.innerText.split(',').join(''));
+
+const totalPop = arrPop.reduce((a, b) => a + b);
+const averagePop = totalPop / arrPop.length;
+
+const totalPopArr = String(totalPop).split('').reverse();
+let totalPopComas = [];
+
+for (let n = 0; n < totalPopArr.length; n++) {
+  totalPopComas.push(totalPopArr[n]);
+
+  if ((n + 1) % 3 === 0 && n !== totalPopArr.length - 1) {
+    totalPopComas.push(',');
+  }
+};
```

Comment:
looks like you used this logic twice - make common function for it

Reply:
and try to rewrite logic with reduce

Reply:
It's impossible, averagePopArr.length !== totalPopArr.length. They couldn't be joined (extra ',' in the average.

Reply:
why you need `averagePopArr.length` here? this comment was about your loop. You can process more easily totalPopArr with `reduce()` method.  

I guess this link can be useful for you
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce
https://stackoverflow.com/questions/61174562/how-reduce-in-javascript-work-if-initial-value-is-array